pkg/rsyslog/build-rsyslog.sh

```diff
@@ -2,6 +2,7 @@
 
 git clone -b v8-stable https://github.com/rsyslog/rsyslog.git
 cd rsyslog || exit
+git checkout ceafdcdfd9c00b97c4f2a57f3d3a2b0b950f76b0
```

Comment:
Does this mean we are locked to one version until we change this?

Reply:
Yes, since we work with diffs, there is no guarantee the diff apply would work if we keep picking latest code. I have seen this happening and that's the reason for fixing commit id.